src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/gateways/Axe.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.scalatestaccessibilitylinter.gateways
+
+import java.io.{ByteArrayInputStream, File}
+import scala.sys.process.Process
+
+object Axe {
+  // requires npm and js from sbt-accessibility-linter
+  def localNpmWrapper(workingDir: String)(html: String): String =
```

Comment:
Not a blocker, but wondered if this functionality could be inlined into the definition of the default Axe linter, since it's not used anywhere else?

Reply:
@matthewmascord I'll have a go at refactoring this and share when it's done to see what you think, this kind of joins up with the comments on the config object

- [ ] refactor stuff into companion objects of linters